exercises/Cargo.toml

```diff
@@ -0,0 +1,286 @@
+# This Cargo.toml is AUTOGENERATED, you shouldn't need to edit it.
+# The `rustling` commands do not rely on this manifest at all, its only purpose
+# is to help editors analyze your code.
+# To regenerate it, run `cargo generate-manifest`.
+[package]
+name = 'exercises'
+version = '0.1.0'
+edition = '2018'
+authors = ['The Rustlings Maintainers']
+publish = false
+
+[[bin]]
```

Comment:
All targets are configured as `[[bin]]`. I'm not sure if this could be problematic for exercises without `main` (maybe some analyzers will complain that is missing?).

Reply:
Errors are produced for each target without a `main` fn.